packages/flutter_tools/test/general.shard/android/android_emulator_test.dart

```diff
@@ -161,20 +151,14 @@ void main() {
             exitCode: 1,
             stderr: errorText,
             stdout: 'dummy text',
-            duration: Duration(seconds: 1),
+            duration: Duration(milliseconds: 1),
```

Comment:
I'm guessing with async code, `Duration.zero == Duration(milliseconds: 1)`?

Reply:
it means "at least zero", but probably more